README.md

```diff
@@ -143,25 +143,14 @@ Please reach out to [@Yatekii](https://github.com/Yatekii)
 
 ### Building
 
-Building requires Rust and Cargo which can be installed [using rustup](https://rustup.rs/). probe-rs also depends on libusb and libftdi. On linux these can be installed with your package manager:
+Building requires Rust and Cargo which can be installed [using rustup](https://rustup.rs/). On linux these can be installed with your package manager:
 
 ```console
 # Ubuntu
-> sudo apt install -y libusb-1.0-0-dev libftdi1-dev libudev-dev libssl-dev
+> sudo apt install -y libudev-dev libssl-dev
```

Comment:
@Yatekii do we still need libssl?

Reply:
I don't think so tbh, but I am not on linux :)
We should also aim to remove libudev as well :)